README.md

```diff
@@ -46,6 +46,16 @@ Click to watch it on YouTube:
 
 ### Installation
 
+Use the following command sequence to install the latest release of the Keptn Lifecycle Toolkit:
+
+```shell
+helm repo add klt https://charts.lifecycle.keptn.sh
+helm repo update
+helm upgrade --install klt klt/klt -n keptn-lifecycle-toolkit-system --create-namespace --wait
```

Comment:
This way we have the following names `keptn-klt-${item}`. Our charts have a name override in the _helper that adds `-klt` to the release name
```suggestion
helm upgrade --install keptn klt/klt -n keptn-lifecycle-toolkit-system --create-namespace --wait
```